test/extensions/filters/http/decompressor/decompressor_filter_integration_test.cc

```diff
@@ -153,4 +155,107 @@ TEST_P(DecompressorIntegrationTest, BidirectionalDecompression) {
       uncompressed_response_length);
 }
 
+/**
+ * Exercises gzip decompression bidirectionally with default configuration.
```

Comment:
I'd replace "default configuration" with something like "configuration using wrong value for Zlib's windowBits".

Reply:
bad copy paste, sorry!